site-manage/datemanager/tool/src/main/java/org/sakaiproject/datemanager/WebAppConfiguration.java

```diff
@@ -41,14 +42,12 @@ public void onStartup(ServletContext servletContext) throws ServletException {
 		servletContext.addListener(new ToolListener());
 		servletContext.addListener(new SakaiContextLoaderListener(rootContext));
 
-		servletContext.addFilter("sakai.request", RequestFilter.class)
-			.addMappingForUrlPatterns(
-				EnumSet.of(DispatcherType.REQUEST, DispatcherType.FORWARD, DispatcherType.INCLUDE),
-				true,
-				"/*");
+		FilterRegistration requestFilterRegistration = servletContext.addFilter("sakai.request", RequestFilter.class);
+		requestFilterRegistration.addMappingForUrlPatterns(EnumSet.of(DispatcherType.REQUEST, DispatcherType.FORWARD, DispatcherType.INCLUDE), true, new String[]{"/import/dates", "/*"});
```

Comment:
request filter is being mapped to "/*" so is "/import/dates" really needed?

Reply:
It's right, with "/*" filter, there is no need to filter "/import/dates"